package.json

```diff
@@ -11,12 +11,16 @@
     "istanbul": "NODE_ENV=test istanbul",
     "lint": "eslint --ext .js,.json .",
     "lint-watch": "esw --watch .",
+    "postrewrite": "yarn run git-hook",
     "precommit": "yarn run git-hook",
     "prepush": "yarn run snyk -- test",
-    "start": "node runner.js",
-    "test": "mocha --recursive",
+    "snyk": "snyk",
+    "snyk-travis": "yarn run snyk -- test --org=nhsuk && yarn run snyk -- monitor --org=nhsuk",
+    "start": "node schedule.js",
+    "test": "snyk test && mocha --recursive",
```

Comment:
I'd suggest `snyk` was not done during `test`. This would also mean the `SYNK_TOKEN` can be removed from the `test-etl` service in `docker-compose-tests.yml`